react/components/ProductImages/components/Carousel/index.js

```diff
@@ -206,12 +208,12 @@ class Carousel extends Component {
       resistanceRatio: slides.length > 1 ? 0.85 : 0,
       renderNextButton: () => (
         <span className={`swiper-caret-next pl7 right-1 ${caretClassName}`}>
-          <Icon id="nav-angle--right" size={24} className="vtex__icon-caret-right" />
+          <Icon id="nav-angle--right" size={24} className={styles.carouselIconCaretRight} />
```

Comment:
I think is better to remove this magic number `24`.

Reply:
Good one. Fixed.